demoapp/run_demo_app.py

```diff
@@ -0,0 +1,116 @@
+"""
+Implements bill summarization, and category and tags generation for all bills (no MGL sections data is available for ~1300 bills)
+All the code implemented here is located in demoapp.py
```

Comment:
But demoapp.py is deleted in this commit. Why is is deleted?

Reply:
I renamed it to demo_app_functions.py. Not sure if that is a good name for it. I can change it back to demo_app.py